mail_texts.json

```diff
@@ -5,8 +5,8 @@
 			"en": "You were approved for {event_name}"
 		},
 		"body": {
-			"de": "Hallo,\r\n\r\nherzlichen Gl\u00fcckwunsch. Du wurdest f\u00fcr {event_name} angenommen. Damit du teilnehmen kannst, muss du nur noch ein paar Fragen beantworten. Das kannst du hier tun: <{change_answer_url}>.\r\n\r\nViele Gr\u00fc\u00dfe,\r\nDas {event_name}-Team",
-			"en": "Hello,\r\n\r\ncongratulations. You were approved for {event_name}. You just need to answer a few more questions so that you can participate. You can do that here: <{change_answer_url}>.\r\n\r\nKind greetings,\r\nthe {event_name}-Team"
+			"de": "Hallo,\r\n\r\nherzlichen Gl\u00fcckwunsch. Du wurdest f\u00fcr {event_name} angenommen. Damit du teilnehmen kannst, muss du nur noch ein paar Fragen beantworten. Das kannst du hier tun: <{change_answer_url}>. {custom_msg}\r\n\r\nViele Gr\u00fc\u00dfe,\r\nDas {event_name}-Team",
```

Comment:
maybe add an `\r\n\r\n` before the custom message as well?

Reply:
Didn't wanna do this here, because we don't need this in case there is no custom message